airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -851,8 +851,9 @@ def run_copy(self,
 
     def run_load(self,
                  destination_project_dataset_table,
```

Comment:
This breaks backward compatibility. This is ok if we target it to Airflow 2.0, and add a line to `CONTRIBUTING.md`

Reply:
@fokko, Exuse me) I can’t catch how it breaks if there are no changes to old args and no logic changes. Just for understanding, seems I miss something. About contributing.md - okay.

Reply:
Well, it changes the order. As a second argument it expects `schema_fields`, and this is moved to the fourth place.

Reply:
@Fokko )) sorry, you right, I forgot about it, I will put it at the end of args list

Reply:
fixed

Reply:
this indeed did break backwards compatibility.  previously (on 1.9) I could run `GoogleCloudStorageToBigQueryOperator` with no autodetect or schema fields, with `source_format='NEWLINE_DELIMITED_JSON'` and it worked fine (presumably doing autodetect?)

now upgrading to 1.10 it breaks since I haven't explicitly specified autodetect.